Methane_MarkDown.Rmd

```diff
@@ -21,53 +21,58 @@ Our first step is to create a core. Hector will use this to pull data from exist
 ```{r  Hector core}
 library(hector)
 library(ggplot2)
+library(dplyr)
 ini_file <- system.file("input/hector_ssp245.ini", package = "hector")
 core <- newcore(ini_file)
 run(core)
 ```
 
 ## Methane Emmisions Data and Graph
 
-After creating the core we then tell hector to provide data for `EMISSIONS_CH4()` between the years of 1795 and 2100. Hector has several options that relate to methane- these include: emissions, concentration, postindustrial, natural, soil lifetime and stratosphere lifetime.
+After creating and running the core we query it for methane-related outputs
+between the years of 1795 and 2100. Hector has several options that relate to methane, including emissions, concentration, postindustrial, natural, soil lifetime and stratosphere lifetime.
 
 ```{r Emissions data frame}
-results<- fetchvars(core, 1795:2100, EMISSIONS_CH4())
+results<- fetchvars(core, 1795:2100, c(EMISSIONS_CH4(),
+                                       CONCENTRATIONS_CH4()))
+shutdown(core)
 head(results)
 ```
 
 ### Plot Results Methane Emissions
 
-Now we have all the things we need to build a simple plot of methane emissions between the years 1795 and 2100. Ggplot2 is helpful for making graphs quickly and simply. We already loaded this package at the beginning of this file. 
+Now we have all the things we need to build a simple plot of methane emissions between the years 1795 and 2100. The `ggplot2` package is helpful for making graphs quickly and simply. We already loaded this package at the beginning of this file. 
 
 ```{r Emissions plot}
-ggplot(data= results, aes(x= year, y= value))+
+results %>% 
+  filter(variable == EMISSIONS_CH4()) %>% 
+  ggplot(aes(x= year, y= value))+
     geom_line(color="blue")+
     labs(x = "Year",
-         y = "Methane Emissions",
+         y = "Methane Emissions (Tg)",
          title = "Methane Emissions 1795-2100 RCP 4.5")
 ```
 
 ## Methane Concentrations
 
-By changing the input `EMISSION_CH4()` to `CONCINTRATIONS_CH4()` we can replicate our code to get different visualized data. To keep things clear these new results are named `results_2`, we also  change the graph name to reflect what is being shown.
+By changing the input `EMISSION_CH4()` to `CONCENTRATIONS_CH4()` we can replicate our code to get different visualized data. To keep things clear these new results are named `results_2`, we also  change the graph name to reflect what is being shown.
 
-```{r Concintration graph}
-results_2<- fetchvars(core, 1795:2100, CONCENTRATIONS_CH4())
-library(ggplot2)
-ggplot(data= results_2, aes(x= year, y= value, ))+
+```{r Concentration graph}
+results %>% 
+  filter(variable == CONCENTRATIONS_CH4()) %>% 
+  ggplot(aes(x= year, y= value, ))+
     geom_line(color= "red")+
     labs(x = "Year",
          y = "Methane Concentration",
          title = "Methane Concentration 1795-2100 RCP 4.5")
 ```
 
-## Compaire Through Plots
+## Comparison Plots
```

Comment:
Can we plot emissions and concentrations on the same graph, since the y axis units are different (Tg vs ppbv)?